File: Lab1.cpp
#include <iostream>

using namespace std;

int main()
{
    string name;
    cout << "What is your name?"<<endl;
    cin>>name;
    cout<<"Hello"<<name<<endl;
    int id;
    cout << "What is your id?"<<endl;
    cin>>id;
    cout<<"Your ID is"<<id<<endl;
  
    int var1;
    cout<<"Var1: "<<endl;
    cin>>var1;
    cout<<"Var2: "<<endl;
    cin>>var2;
    int sum;
    sum = var1+var2;
    int diff;
    diff =var1-var2;
    int prod;
    prod = var1*var2;
    cout<<"var1: " <<var1 <<"\n";
    cout<<"var2: " <<var2 <<"\n";
    cout<<"Sum: " <<sum <<"\n";
    cout<<"Diff: " <<diff <<"\n";
    cout<<"Prod: " <<prod <<endl;

  

    cout<<"*"<<"\n"<<"**"<<"\n"<<"***"<<"\n"<<"**"<<"\n"<<"*"<<endl;
    return 0;
}

File: lab-1/Lab1.py
name=input("What is your name? ")
print("Hello" , name)
id=input("What is your Student ID? ")
print("Your ID is " , id)

print("Enter var 1: ")
var1=input()
var1=int(var1)
print("Enter var2: ")
var2=input()
var2=int(var2)
sum=var1+var2
print(sum)
diff=var1-var2
print(diff)
prod=var1*var2
print(prod)

name=input("What is your name? ")
gradeLab=input("What is your lab grade? ")
grade=int(gradeLab)
lab=grade*25/100
gradeMidterm=input("What is your midterm grade? ")
grade=int(gradeMidterm)
midterm=grade*35/100
gradeFinal=input("What is your grade? ")
grade=int(gradeFinal)
final=grade*40/100
score=lab+final+midterm
print("Name: ", name)
print("Lab: ", lab)
print("Midterm: ", midterm)
print("Final:", final)
print("Last Score: ", score)

print("*\n**\n***\n**\n*")
